### PR TITLE
Remove wrong usage in usingdocker.md

### LINKED ISSUE #13235: docker command without argument doesn't show help
usingdocker.md says "Try typing Docker followed with a [command] to see the usage for that command:",
$ docker attach
Help output . . .

But it will not show help without additional arguments:
$ docker attach
docker: "attach" requires 1 argument. See 'docker attach --help'.

### PR DESCRIPTION
**Pull request by hqhq**
Fixes: https://github.com/docker/docker/issues/13235

Signed-off-by: Qiang Huang h.huangqiang@huawei.com

### CODE REVIEW

File: docs/sources/userguide/usingdocker.md [hunk 1]
```diff
@@ -68,13 +68,8 @@ You will see a list of all currently available commands.
...
 You can also zoom in and review the usage for specific Docker commands.
 
-Try typing Docker followed with a `[command]` to see the usage for that
-command:
-
-    $ docker attach
-    Help output . . .
-
-Or you can also pass the `--help` flag to the `docker` binary.
+Try typing Docker command followed with a `--help` to see the usage for
```
Comment (thaJeztah) [inline docs/sources/userguide/usingdocker.md]: Try typing *a* Docker command (missing 'a' before Docker)

**Comment by thaJeztah** [comment]
One small note, but otherwise LGTM (but this may become obsolete once https://github.com/docker/docker/pull/11858 is merged).

ping @fredlf @jamtur01 @moxiegirl @SvenDowideit

File: docs/sources/userguide/usingdocker.md (hunk 1 above)
Comment (jamtur01) [inline docs/sources/userguide/usingdocker.md]: Or add "flag"

**Comment by hqhq** [comment]
@thaJeztah Updated. I don't think it will be obsolete even https://github.com/docker/docker/pull/11858 is merged. Because type docker command without any arguments is the way to see help is still not true.

File: docs/sources/userguide/usingdocker.md [hunk 2]
```diff
@@ -68,13 +68,8 @@ You will see a list of all currently available commands.
...
 You can also zoom in and review the usage for specific Docker commands.
 
-Try typing Docker followed with a `[command]` to see the usage for that
-command:
-
-    $ docker attach
-    Help output . . .
-
-Or you can also pass the `--help` flag to the `docker` binary.
+Try typing a Docker command followed with flag `--help` to see the usage for
```
Comment (moxiegirl) [inline docs/sources/userguide/usingdocker.md]: This sentence is missing an article.  Also, well, the whole section reads a bit oddly. I wonder if you'd be willing to expand the scope a bit to clean up a few things:

-----> https://gist.github.com/moxiegirl/fb4d11255b598884f892

### Get Docker command help

You can display the help for specific Docker commands. The help details the
options and their usage. To see a list of all the possible commands, use the
following:

		$ docker --help

To see usage for a specific command, specify the command with the `--help` flag:

    $ docker attach --help

		Usage: docker attach [OPTIONS] CONTAINER

		Attach to a running container

			--help=false        Print usage
			--no-stdin=false    Do not attach STDIN
			--sig-proxy=true    Proxy all received signals to the process

For further details and examples of each command, see the  [command
reference](/reference/commandline/cli/) in this guide.

Comment (fredlf) [inline docs/sources/userguide/usingdocker.md]: +1 to @moxiegirl 's suggested change/addition.

Comment (thaJeztah) [inline docs/sources/userguide/usingdocker.md]: Agreed, that's much better. I didn't want to diverge too much from the original text, but this is better :)

**Comment by hqhq** [comment]
@moxiegirl Thanks for your suggestion, updated.

**Comment by thaJeztah** [comment]
thanks! LGTM

ping @fredlf @jamtur01 @moxiegirl @SvenDowideit

**Comment by moxiegirl** [comment]
LGTM

**Pull request merged**